script/test

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Install cld2-cffi package with an additional CFLAG
+CFLAGS="-Wno-narrowing" pip3 install --quiet cld2-cffi
+
+# Install requirements and test requirements
+pip3 install --quiet --requirement requirements.txt
```

Comment:
Move this to `script/bootstrap`? And drop the explicit references to `pip3` and `python3` everywhere?